sql/table.cc

```diff
@@ -7763,6 +7766,7 @@ void TABLE::vers_update_fields()
   }
 
   vers_end_field()->set_max();
+  bitmap_set_bit(read_set, vers_end_field()->field_index);
 }
```

Comment:
What are these two `bitmap_set_bit` for?

Reply:
Because of this access (it checks assertion):
```c++
if (versioned() && !vers_end_field()->is_max())
```
Though not required for `vers_start_field()`, this was made for future similar cases: possibility to read value from field which was set earlier. Should not influence performance, I guess?

Reply:
I think it shouldn't. But generally, better not to mark fields that aren't needed. Not a big deal, if we'll need to check `vers_start_field()` in the future, it'll be easy to add `bitmap_set_bit`. One cannot miss it, that's what an assert is for :smile:

Reply:
Got it.